students/Briadean/lesson06/Mailroom4.py

```diff
@@ -0,0 +1,159 @@
+#!/usr#!/usr/bin/env python3
+
+import sys
+
+
+donors_dict = {"Retsuko Rarecho": [100.00, 25.00, 75.00, 200.00],
+               "Fenneko Inoue": [10.00, 200.00, 75.00],
+               "Haida Kato": [15.00, 15.00, 15.00, 15.00, 15.00],
+               "Gori Tsuruta": [25.00, 30.00, 60.00, 90.00],
+               "Washimi Koiwasaki": [10.00, 10.00, 10.00, 35.00, 15.00]}
+
+
+def main_menu_choice():
+    """Display main menu and collect user input"""
+    user_choice = input("\n1: Send a Thank You\n"
+                        "2: Generate a Report\n"
+                        "3: Send Letters to Everyone\n"
+                        "4: Exit the Mailroom\n"
+                        "Please enter your selection from the options above >> ")
+    return user_choice
+
+
+def thank_you_menu():
+    """Display Thank You menu and collect user input"""
+    user_choice = input("\nPlease enter a donor name, "
+                        "'List' to see a list of previous donors, "
+                        "or 'Menu' to return to the menu. >> ").title()
+    while True:
+        if user_choice == "List":
+            list_donors()
+            # Is there a better way to get back to user_choice = input? Break/return/continue all go back to main menu
```

Comment:
Good question. You could also put the original user_choice = input(blahblah) inside of the True block and above the first "if" (on line 29) to cut down on this repetition.